docs/resources/Subscription.md

```diff
@@ -0,0 +1,72 @@
+| sku_ids              | array of snowflakes | List of SKUs subscribed to                                                                                                                 |
+| entitlement_ids      | array of snowflakes | List of entitlements granted for this subscription                                                                                         |
+| current_period_start | ISO8601 timestamp   | Start of the current subscription period                                                                                                   |
+| current_period_end   | ISO8601 timestamp   | End of the current subscription period                                                                                                     |
+| status               | SubscriptionStatus  | Current status of the subscription                                                                                                         |
+| canceled_at          | ISO8601 timestamp?  | When the subscription was canceled                                                                                                         |
+| country?             | string              | ISO3166-1 alpha-2 country code of the payment source used to purchase the subscription. Missing unless queried with a private OAuth scope. |
+
+The start of a subscription is determined by its ID. When the subscription renews, its current period is updated.
+
+If the user cancels the subscription, the `canceled_at` timestamp reflects when they did so. The subscription will enter the `ENDING` status.
+
+### Subscription Example
+
+```json
+{
+  "id": "1278078770116427839", 
+  "user_id": "1088605110638227537", 
+  "sku_ids": ["1158857122189168803"], 
+  "entitlement_ids": [], 
+  "current_period_start": "2024-08-27T19:48:44.406602+00:00", 
+  "current_period_end": "2024-09-27T19:48:44.406602+00:00", 
+  "status": 0, 
+  "canceled_at": null
+}
+```
+
+### Subscription Statuses
+
+| Type     | Value | Description                                         |
+|----------|-------|-----------------------------------------------------|
+| ACTIVE   | 1     | The subscription is active and scheduled to renew.  |
+| ENDING   | 2     | The subscription is active but will not renew.      |
+| INACTIVE | 3     | The subscription is inactive and not being charged. |
+
+> info
+> Subscription status should not be used to grant perks. Use [entitlements](#DOCS_RESOURCES_ENTITLEMENT/entitlement-object) as an indication of whether a user should have access to a specific SKU. See our guide on [Implementing App Subscriptions](#DOCS_MONETIZATION_IMPLEMENTING_APP_SUBSCRIPTIONS) for more information.
+
+Subscriptions can start at and change between any of these statuses at any time.
+
+Status is *not* related to the current period. A subscription could be `ACTIVE` outside its current period or `INACTIVE` within its current period. 
```

Comment:
how?? this doesn't make sense at first glance so could help to be more explicit about why/when this isn't the case